packages/widget.vite-plugin/src/plugin.ts

```diff
@@ -231,6 +234,24 @@ export function FoundryWidgetVitePlugin(options: Options = {}): Plugin {
               ?? {},
           };
           widgetConfigManifest.widgets[chunk.name] = widgetConfig;
+          entrypointImports[chunk.name] = chunk.imports;
+        } else {
+          // Check if it's an imported chunk, since any CSS files we will need to put on the page for them
+          // JS files will get imported on their own
+          for (
+            const [entrypointName, imports] of Object.entries(entrypointImports)
+          ) {
+            if (
+              imports.includes(chunk.fileName)
+              && chunk.viteMetadata?.importedCss.size
+            ) {
+              widgetConfigManifest.widgets[entrypointName].entrypointCss?.push(
+                ...[...chunk.viteMetadata.importedCss].map((css) => ({
+                  path: css,
+                })),
```

Comment:
Do we know how many entrypoint's this produces? E.g. is it one per entrypoint / dynamic import etc, rather than one per literal CSS file?

Reply:
ah fair, it could be duplicated 🤔 will fix